src/Symfony/Component/Serializer/Normalizer/DateTimeZoneNormalizer.php

```diff
@@ -57,7 +57,7 @@ public function denormalize($data, $class, $format = null, array $context = [])
         try {
             return new \DateTimeZone($data);
         } catch (\Exception $e) {
-            throw new NotNormalizableValueException($e->getMessage(), $e->getCode(), $e);
+            throw new NotNormalizableValueException($e->getMessage().sprintf('ProTip: The DateTime for class "%s" seems wrong ("%s" given).', $class, $data), $e->getCode(), $e);
```

Comment:
missing space before "Pro-tip" - there are more such spaces missing in the PR
and pro-tip is not appropriate I think btw